Dockerfile

```
@@ -0,0 +1,44 @@
+#FROM ocm-container
+FROM fedora:latest
+
+ENV I_AM_IN_CONTAINER="I-am-in-container"
+
+RUN yum -y install \
+    bash-completion \
+    findutils \
+    fzf \
+    git \
+    golang \
+    jq \
+    make \
+    procps-ng \
+    rsync \
+    sshuttle \
+    vim-enhanced \
+    wget \
+    && yum clean all;
```

You can save more disk space by removing the yum cache as well:

`&& yum clean all && rm -r /var/cache/yum/`

---

Nice, we should add that in!

```suggestion
    && yum clean all \
    && rm -r /var/cache/yum;
```